### PR TITLE
Some minor fixes

### PR DESCRIPTION
**Pull request by dvilchez**
I'm using it in my project and I have to make some fixes to get it work properly.

### CODE REVIEW

**Comment by pughpugh** [comment]
Ta, thanks @dvilchez

File: coffee/react-countdown-clock.coffee [hunk 1]
```diff
@@ -23,13 +23,15 @@ module.exports = React.createClass
     seconds: @props.seconds
 
   componentWillReceiveProps: (props) ->
-    @setState seconds: props.seconds
-
-  componentWillMount: ->
     @_setScale()
```
Comment (pughpugh) [inline coffee/react-countdown-clock.coffee]: I notice the demo page now accelerates after each iteration.

In hindsight I'm not sure this works. `_setScale` needs to be called after the `setState seconds: props.seconds` has been called. Even if you move the ordering around, I think the asynchronous nature of setState still messes this up.

**Comment by dvilchez** [comment]
I can't notice it, but after reviewing the code I think it would be more simple if it wouldn't set the state due to the fact that it doesn't need that render method was called to redraw the canvas.

**Comment by pughpugh** [comment]
Do you mean - why set state if it's not intended to recall render? If so, I think I agree.

**Comment by dvilchez** [comment]
Yes, That's it!

**Pull request merged**